src/migration-scripts/ids/1-to-2

```diff
@@ -0,0 +1,30 @@
+# Copyright 2025 VyOS maintainers and contributors <maintainers@vyos.io>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+# T: Migrate threshold and add new threshold types
+
+from vyos.configtree import ConfigTree
+
+# The old 'service ids' path was only used for FastNetMon
+# Suricata is in 'service suricata',
+# so this isn't an overreach
+base = ['service', 'ids']
+
+def migrate(config: ConfigTree) -> None:
+    if not config.exists(base):
+        # Nothing to do
+        return
+    else:
+        config.delete(base)
```

Comment:
Do we need a `1-to-2` migration, or can we make this change in the `0-to-1` migration instead?
Because in my understanding, if you add a new migration, it should be declared/marked here https://github.com/vyos/vyos-1x/blob/f06087155cb6c9e8d7f39d1ac41eef8f1a2cf288/interface-definitions/include/version/ids-version.xml.i#L2

Reply:
Good catch, that was a file missing from the PR! I've included it now.